src/server/game/Chat/Chat.cpp

```diff
@@ -16,6 +16,7 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <boost/algorithm/string/replace.hpp>
```

Comment:
include order should be
* Main header for the source (chat.h)
* all other TC includes
* 3rd party lib includes
* stl includes
* c includes